aiida_lsmo/workchains/sim_annealing.py

```diff
@@ -36,15 +36,17 @@ def get_molecule_from_restart_file(structure_cif, molecule_folderdata, input_dic
 
     # Get the atom types of the molecule (ASE accepts only atomic eilements as "symbols")
     ff_data_molecule = load_yaml()[molecule_dict['name']][molecule_dict['forcefield']]
-    symbolsff = [x[0].split('_')[0] for x in ff_data_molecule['atomic_positions']]
+    symbolsff = [x[0] for x in ff_data_molecule['atomic_positions']]
```

Comment:
what changed that one now needs this change?

Reply:
was this unnecessary before and now would fail with the `_` in the `DMA`?

Reply:
Before, I was extracting the molecule-specific labels. This works fine for e.g. CO2, where it extracts C from `C_co2` and O from `O_co2`. But i fails if the FF atoms are not defined as chemical symbols e.g. N1 in `N1_dma`. Instead, I'm now directly using the chemical symbol as defined in the `pseudo_atoms` which works fine for FF where the same element has different definitions and those different labels. O.c. this will still fail if we use it for united-atom FF

Reply:
no, I'm not an expert in this code base but it looks reasonable to be.

Thanks again for maintaining it and sorry for the late reply